specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/inference.json

```
@@ -0,0 +1,1249 @@
+      "get": {
+        "tags": [
+          "MachineLearningInferenceDeployments"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "operationId": "MachineLearningInferenceDeployments_ListByInferenceEndpoint",
+        "description": "Gets Inference Deployments.",
+        "x-ms-examples": {
+          "Get Deployments": {
+            "$ref": "./examples/getDeployments.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/PaginationParameter"
+          },
+          {
+            "$ref": "#/parameters/InferenceEndpointNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The response includes a paginated array of Machine Learning Inference Deployments.",
+            "schema": {
+              "$ref": "#/definitions/PaginatedInferenceDeploymentsList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the request failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/inferenceEndpoints/{inferenceEndpointName}/deployments/{deploymentName}": {
+      "get": {
+        "tags": [
+          "MachineLearningInferenceDeployments"
+        ],
+        "operationId": "MachineLearningInferenceDeployment_Get",
+        "description": "Get a Inference Deployment by name.",
+        "x-ms-examples": {
+          "Get Deployment": {
+            "$ref": "./examples/getDeployment.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/InferenceEndpointNameParameter"
+          },
+          {
+            "$ref": "#/parameters/InferenceDeploymentNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/InferenceDeploymentResource"
```

Should we fully qualify?
```suggestion
              "$ref": "#/definitions/InferenceEndpointDeploymentResource"
```